.travis.yml

```diff
@@ -17,6 +17,7 @@ env:
     - TASK=r_test
     # python package test
     - TASK=python_test
+    - OMP_NUM_THREADS=3 TASK=python_omp_test
```

Comment:
Indeed, I verified that setting `nthread` does enable multithreading. I will move the new test to `python_test` group.